packages/cascara/src/modules/DataEmail/DataEmail.js

```diff
@@ -29,26 +30,35 @@ const DataEmail = ({
 }) => {
   const { isEditing, formMethods } = useContext(ModuleContext);
 
+  // do not add aria label if tag is used
+  const conditionalLabelProps = getConditionalLabelProps(label, isLabeled);
+
   const renderEditing = (
-    <label htmlFor={label}>
-      {label && isLabeled && <span className={styles.LabelText}>{label}</span>}
+    <>
+      {label && isLabeled && (
+        <label htmlFor={attribute || label}>
+          <span className={styles.LabelText}>{label || attribute}</span>
+        </label>
+      )}
       <Input
+        {...conditionalLabelProps}
         {...rest}
-        aria-label={label}
         className={styles.Input}
         defaultValue={value}
-        id={label}
+        id={attribute || label}
         name={attribute || label}
         ref={formMethods?.register}
         type='email'
       />
-    </label>
+    </>
```

Comment:
It looks like we are changing this markup everywhere and now making the label a sibling of the inputs instead of wrapping them. This has accessibility implications as well as style implications based on how the nesting of the styles are defined. We need to maintain the label tag as wrapping the input.

Reply:
Thanks for catching this, we definitely don not want that. This has been solved by the new `LabeledModule` helper component, in which we abstract this logic away and output the module as `children` (wrap).